### PR TITLE
Reordering content of "why care about reproducibility" chapter

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

The language from the original content by F. Markowetz's paper did not seem very clear. With the current content I have tried to remove jargon and add a few missing bits.

### List of changes proposed in this PR (pull-request)

* New bullet points and content added
* A few new references added to the bibliography
* An infographic was added to align with the content

### What should a reviewer concentrate their feedback on?

- [ ] Content, image and citations
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.

### CODE REVIEW

File: book/content/reproducibility/02/whycare.md [hunk 1]
```diff
@@ -1,14 +1,40 @@
-# Why you should care about reproducibility
+# Advantages of working reproducibly
 
-Markowetz highlights five reasons to work reproducibly (Markowetz, 2015):
+We have so far discussed the different aspects of reproducible research that are beneficial for the scientific community. In this chapter, we will share some less obvious aspects of working reproducibly for individual researchers and teams.
```
Comment (sgibson91) [inline book/content/reproducibility/02/whycare.md]:
```suggestion
We have so far discussed the different aspects of reproducible research that are beneficial for the scientific community.
In this chapter, we will share some less obvious aspects of working reproducibly for individual researchers and teams.
```

File: book/content/reproducibility/02/whycare.md [hunk 2]
```diff
@@ -1,14 +1,40 @@
...
-- **Building your reputation**: Putting in effort to make your research reproducible shows that you are a careful researcher and makes your research results more robust.
+![Why care about working reproducibly](../../figures/reasons_reproducibility.png)
 
-Papers whose underlying data is available get cited more often (Piwowar, Day & Fridsma, 2007; Piwowar & Vision, 2013).
-All research outputs that are shared can be built upon more easily by others and in some cases, others following up on your work might lead to new collaborations.
-Other benefits of working openly are covered in our [Open Research](../open_research/open_research) chapter.
+**1. Track a complete history of your research**
+
+Reproducible research must contain a complete history and narrative (also known as [Provenance](https://en.wikipedia.org/wiki/Provenance)) of the project planning and development process. This includes information on the data, tools, methods, codes, and documentation used in the research project. By storing a complete track-record of our work, we can ensure research sustainability, fair citation/acknowledgment, and usefulness of our and others' work in our research fields.
```
Comment (sgibson91) [inline book/content/reproducibility/02/whycare.md]:
```suggestion
Reproducible research must contain a complete history and narrative (also known as [Provenance](https://en.wikipedia.org/wiki/Provenance)) of the project planning and development process. This includes information on the data, tools, methods, codes, and documentation used in the research project.
By storing a complete track-record of our work, we can ensure research sustainability, fair citation/acknowledgment, and usefulness of our and others' work in our research fields.
```

File: book/content/reproducibility/02/whycare.md [hunk 3]
```diff
@@ -1,14 +1,40 @@
...
+
+**2. Facilitate collaboration and review process**
+
+By designing reproducible workflows and sharing them with the different components of our research project, we can allow our collaborators to develop an in-depth understanding of our work. This encourages them to review different parts of our work, test our code, propose useful changes and make thoughtful contributions to develop our project further.
```
Comment (sgibson91) [inline book/content/reproducibility/02/whycare.md]:
```suggestion
By designing reproducible workflows and sharing them with the different components of our research project, we can allow our collaborators to develop an in-depth understanding of our work.
This encourages them to review different parts of our work, test our code, propose useful changes and make thoughtful contributions to develop our project further.
```

File: book/content/reproducibility/02/whycare.md [hunk 4]
```diff
@@ -1,14 +1,40 @@
...
+
+**3. Publish validated research and avoid misinformation**
+
+Lack of reproducibility is one of the major factors that lead to paper retractions (source [Retraction Watch](https://retractionwatch.com/)). The best-known analyses of scientific literatures in psychology {% cite Begley2012 %} and cancer biology {% cite aac47162015 %} found the reproducibility rates of their research output of around 40% and 10%, respectively. By working reproducibly, we can develop validated research work, avoid misinformation that can limit replicability of our work and publish accurate research outputs. This aspect does not only support the validity of the current work, but any future studies that are based on reproducible research {% cite MozillaScienceLab %}.
```
Comment (sgibson91) [inline book/content/reproducibility/02/whycare.md]:
```suggestion
Lack of reproducibility is one of the major factors that lead to paper retractions (source [Retraction Watch](https://retractionwatch.com/)).
The best-known analyses of scientific literatures in psychology {% cite Begley2012 %} and cancer biology {% cite aac47162015 %} found the reproducibility rates of their research output of around 40% and 10%, respectively.
By working reproducibly, we can develop validated research work, avoid misinformation that can limit replicability of our work and publish accurate research outputs.
This aspect does not only support the validity of the current work, but any future studies that are based on reproducible research {% cite MozillaScienceLab %}.
```

File: book/content/reproducibility/02/whycare.md [hunk 5]
```diff
@@ -1,14 +1,40 @@
...
+
+**4. Write your paper efficiently**
+
+Well documented analyses help us maintain easy access to all the results generated within a project that can be written up efficiently. If working in a team, collaborators can easily get recognition such as authorship for their contributions. Furthermore, by availing the underlying dataset and methods we can easily comply with the highest-level journal guidelines.
```
Comment (sgibson91) [inline book/content/reproducibility/02/whycare.md]:
```suggestion
Well documented analyses help us maintain easy access to all the results generated within a project that can be written up efficiently.
If working in a team, collaborators can easily get recognition such as authorship for their contributions.
Furthermore, by availing the underlying dataset and methods we can easily comply with the highest-level journal guidelines.
```

File: book/content/reproducibility/02/whycare.md [hunk 6]
```diff
@@ -1,14 +1,40 @@
...
+
+**5. Get credits for your work fairly**
+
+Researchers who use the principles of reproducibility in developing their projects publish their work with underlying data and methods. Such papers get cited more often as their research outcome can be broadly replicated and trusted.
```
Comment (sgibson91) [inline book/content/reproducibility/02/whycare.md]:
```suggestion
Researchers who use the principles of reproducibility in developing their projects publish their work with underlying data and methods.
Such papers get cited more often as their research outcome can be broadly replicated and trusted.
```

File: book/content/reproducibility/02/whycare.md [hunk 7]
```diff
@@ -1,14 +1,40 @@
...
+
+**6. Ensure continuity of your work**
+
+By following guidelines for reproducibility, we can easily communicate our work with different stakeholders such as our supervisors, funders, reviewers, students, and potential collaborators. This aspect of reproducibility increases the usefulness of our research by enabling others to easily build on your results, and re-use your research materials {% cite MozillaScienceLab %}. Progress of such projects can easily be tracked and continued - either by others in your laboratory, or yourself if you want to build on your own work after a longer period {% cite Markowetz2015 %}. This ensures the continuity of a research idea and can even find their applications in different contexts.
```
Comment (sgibson91) [inline book/content/reproducibility/02/whycare.md]:
```suggestion
By following guidelines for reproducibility, we can easily communicate our work with different stakeholders such as our supervisors, funders, reviewers, students, and potential collaborators.
This aspect of reproducibility increases the usefulness of our research by enabling others to easily build on your results, and re-use your research materials {% cite MozillaScienceLab %}.
Progress of such projects can easily be tracked and continued - either by others in your laboratory, or yourself if you want to build on your own work after a longer period {% cite Markowetz2015 %}.
This ensures the continuity of a research idea and can even find their applications in different contexts.
```

**Comment by malvikasharan** [comment]
Thank you so much for the review @sgibson91 :dancer: And thank you for being patience - I need to remember to line breaks.

**Comment by KirstieJane** [comment]
Great stuff - thank you @malvikasharan & @sgibson91 🚀

**Pull request merged**